modules/api/src/main/java/org/apache/ignite/rest/RestAuthConfig.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.rest;
+
+import java.util.Collections;
+import java.util.List;
+
+/** Configuration of REST authentication. */
+public class RestAuthConfig {
```

Comment:
`Auth` is ambiguous: is it `Authorization` or `Authentication` or both?

Let's rename this class and everything related to include the full term: `RestAuthenticationConfig`.

Reply:
done